### PR TITLE
Multiple analyzers

### PR DESCRIPTION
**Pull request by smacker**
The first point in #20 

Not sure if I understood it correctly.

There is a new target `make dry-run` to run the server.

### CODE REVIEW

**Comment by bzz** [comment]
Seems to address both, 1 and 3 from #20 

`dry-run` idea is 👍 as that is something that can be used/refactored to `-mode cli` for #40

File: cmd/lookout/serve.go [hunk 1]
```diff
@@ -151,22 +175,36 @@ func (c *ServeCommand) handlePR(e *lookout.ReviewEvent) error {
...
-		logger.Infof("no comments were produced")
-		return nil
+	var comments []*lookout.Comment
+
+	for name, a := range c.analyzers {
+		aLogger := logger.With(log.Fields{
+			"analyzer": name,
+		})
+
+		resp, err := a.NotifyReviewEvent(context.TODO(), e)
```
Comment (bzz) [inline cmd/lookout/serve.go]: This would block on every analyzer, waiting for response, before sending a request to the next one, wouldn't it?

Comment (smacker) [reply cmd/lookout/serve.go]: true. Thanks! I was still in a mindset of sdk where everything should be synchronous. 

I update PR using WaitGroup & Mutex.
Though we might use another solution later.

File: cmd/lookout/serve.go [hunk 2]
```diff
@@ -21,10 +25,21 @@ func init() {
...
+	Addr string
+}
+
+// Config is a server configuration
+type Config struct {
+	Analyzers []AnalyzerConfig
+}
+
 type ServeCommand struct {
+	ConfigFile  string `long:"config" short:"c" default:"config.yml" env:"CONFIG_FILE" description:"path to configuration file"`
```
[the remark this answers is not in the document]
Comment (smacker) [reply cmd/lookout/serve.go]: currently, none of the other env vars use this prefix. Should we add it everywhere? What about dummy analyzer and serve commands?

**Comment by smacker** [comment]
rebased on master

File: .gitignore [hunk 3]
```diff
@@ -14,8 +14,13 @@
...
+<<<<<<< HEAD
 # build results
 /build
 
 # dependencies
 /protoc
+=======
+# Configuration
+/config.yml
+>>>>>>> 88a0c8b... support multiple analyzers
```
Comment (bzz) [inline .gitignore]: Seems like artefact of rebase left by accident

File: Makefile [hunk 4]
```diff
@@ -83,3 +84,9 @@ $(DUMMY_BIN):
 clean-sdk:
 	rm -f $(DUMMY_BIN)
 	rm -f $(LOOKOUT_BIN)
+
+.PHONY: dry-run
+dry-run: $(CONFIG_FILE)
+	go run cmd/lookout/*.go serve --dry-run github.com/src-d/lookout
+$(CONFIG_FILE):
+	cp "$(CONFIG_FILE).tpl" $(CONFIG_FILE);
```
Comment (bzz) [inline Makefile]: empty line?

**Comment by smacker** [comment]
Thanks! Fixed. Waiting for CI & Merge.

**Pull request merged**